### DISCUSSION TITLE
Flag for displaying handles

### DISCUSSION THREAD

**Discussion by MarioMastr** [qnblackcat/uYouPlus#1009] [General]
### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### Is there an existing issue/question for this?

- [X] I have searched the existing issues

### Do you think this is a bug?

❌ No, I don't think this is a bug. I will explain below

### My question

Is there anyone here that knows the flag for displaying handles in the comment section instead of regular usernames? I had it once and then it went away.

**Comment by arichornlover** [comment]
Hey ummm I’m very positive that this is a client-side change, I was using the oldest version of YouTube and handles were still shown in the comments section which is surprising to see in a very old version. but If there is a flag I’ll see if I can find it.
